library/src/main/java/com/phillipcalvin/iconbutton/IconButton.java

```diff
@@ -18,6 +18,7 @@
 
     private enum DrawablePositions {
         NONE,
+        BOTH,
```

Comment:
Does it make sense to call this `BOTH_LEFT_AND_RIGHT` in case top and bottom support is added too?